### PR TITLE
Add v1.0.0 utils methods testcase

### PR DESCRIPTION
**Pull request by HuskyHsu**
# TICKET

# ISSUE

# DESCRIPTION

# CHANGES

### CODE REVIEW

File: src/testcase/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+const testCases: TestCase[] = testCaseInput.map((item, i) => {
+  return {
+    input: item as BodyInfo,
+    expected: testCaseExpected[i] as Expected,
+  };
+});
+
+export const testFn = (getAns: Function, fn: Function) => () => {
```
Comment (ashramwen) [inline src/testcase/index.ts:17]:
```suggestion
export const testFn = (getAns: (expected: Expected) => string, fn: Formula) => () => {
```

File: src/testcase/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import testCaseInput from './input.json';
+import testCaseExpected from './expected.json';
+import { BodyInfo, Expected } from '../models';
```
Comment (ashramwen) [inline src/testcase/index.ts:3]:
```suggestion
import { BodyInfo, Expected, Formula } from '../models';
```

File: src/testcase/index.ts (hunk 1 above)
Comment (HuskyHsu) [reply src/testcase/index.ts]: fixed

File: src/testcase/index.ts (hunk 2 above)
Comment (HuskyHsu) [reply src/testcase/index.ts]: fixed

**Review by ashramwen — APPROVED** [review]

**Pull request merged**